src/clj/schema/experimental/generators.clj

```diff
@@ -116,6 +116,7 @@
     s/Symbol (generators/fmap (comp symbol name) generators/keyword)
     Object generators/any
     s/Any generators/any
+    s/Uuid (generators/fmap (fn [_] (java.util.UUID/randomUUID)) generators/int)
```

Comment:
This isn't correct -- you need to make sure you're using the random number generator internal to generators.  (They are supposed to be deterministic given the outer seed, and using a different RNG subverts that).  This thread shows some examples (https://gist.github.com/jcf/7a91c9473b57b1834704) although there's some question of whether they generate valid UUIDs -- I don't know too much about them.